doc/json-schemas/build-info.schema.json

```diff
@@ -0,0 +1,83 @@
+{
+  "$schema": "https://json-schema.org/draft/2019-09/schema",
+  "type": "object",
+  "properties": {
+    "cabal-version": {
```

Comment:
`cabal-install` version?

Reply:
no, this is intended to be the `lib:Cabal` version.

Reply:
plan.json has `"cabal-lib-version"` for that so we should probably be consistent with that here.

Reply:
Updated